test_all.sh

```diff
@@ -48,7 +48,7 @@ eval set -- "$OPTS"
 while true ; do
   case "$1" in
     -b | --benchmarks )
-         [ -d "$2" ] || usage "Benchmarks directory [$2] not found."
+         [ -d "$2" ] || usage "Fuck you and Benchmarks directory [$2] not found."
```

Comment:
That's a little too aggressive :stuck_out_tongue:

Reply:
Oh no I didn't know why I wrote this I must be drunk.